### PR TITLE
Always allow teacher to see student level

### PR DESCRIPTION
**Pull request by dmcavoy**
Instead of blocking the whole level if a student has never visited a level we want to allow a teacher to be able to leave feedback for that student while still knowing the student has not started the level yet.

This only shows up on levels that don't have a channel id or if you are on a level that does not need a channel id - is not free play - and the level progress number is undefined.

# Before

Screen Shot 2019-06-12 at 2 46 37 PM

# After

Screen Shot 2019-06-12 at 2 43 45 PM

### CODE REVIEW

File: apps/src/StudioApp.js [hunk 1]
```diff
@@ -3154,6 +3154,25 @@ StudioApp.prototype.isResponsiveFromConfig = function(config) {
...
+
+  if (
+    ['Gamelab', 'Applab', 'Weblab', 'Spritelab'].includes(config.levelGameName)
+  ) {
+    return config.readonlyWorkspace && !config.channel;
+  } else if (!config.level.freePlay) {
+    return (
+      config.readonlyWorkspace &&
+      progress.levelProgress[progress.currentLevelId] === undefined
+    );
```
Comment (joshlory) [inline apps/src/StudioApp.js:3172]: What does this do?

**Review by joshlory — APPROVED** [review]
🎉

**Review by maddiedierker — APPROVED** [review]
🥇such great work, dani! i'm really excited for this change

File: apps/src/StudioApp.js (hunk 1 above)
Comment (dmcavoy) [reply apps/src/StudioApp.js:3172]: This is just another condition when we know that a level has not been started yet by a student. In levels that are not free play if a student has not ever run the program the levelProgress will be undefined.

**Pull request merged**